opentelemetry/chart/templates/_helpers.tpl

```diff
@@ -8,24 +8,24 @@ Generic plugin name
 {{/* Generate plugin specific labels */}}
 {{- define "plugin.labels" -}}
 plugindefinition: opentelemetry
-{{- if .Values.global.commonLabels }}
-{{ tpl (toYaml .Values.global.commonLabels) . }}
+{{- if .Values.commonLabels }}
+{{ tpl (toYaml .Values.commonLabels) . }}
 {{- end }}
 {{- end }}
 
 {{/* Generate prometheus specific labels */}}
 {{- define "plugin.prometheusLabels" }}
-{{- if .Values.openTelemetry.prometheus.additionalLabels }}
-{{- tpl (toYaml .Values.openTelemetry.prometheus.additionalLabels) . }}
+{{- if .Values.opentelemetry.prometheus.additionalLabels }}
```

Comment:
What's the reason for uncamel-casing?

Reply:
I chose this for the sake of consistency. Plenty of other references used all lowercase, the otel-operator dependency is also named in all lowercase, the lowercase name is the same for the directory of the plugin + the reference for PRs, issues etc.

Reply:
Turns out that I was a bit too eager here. I had missed quite a few related references of this particular name. I will revert the casing and then in the future potentially having a dedicated PR for dealing with all cascading/depending references.